src/components/ContentSidebar/ActivitySidebar.js

```diff
@@ -438,29 +459,59 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             activityFeedError,
             currentUserError,
         } = this.state;
+
+        // Since this is the first render, we know that ActivityFeed will finally be lazy loaded in!
+        if (!hasAFJSBegunLoading) {
+            Timer.mark(ACTIVITY_FEED_START_TAG);
+            hasAFJSBegunLoading = true;
+        } else if (isAFJSLoaded) {
+            // If the JS is loaded, then we can stop some timers!
+            if (!hasRenderMetricBeenRecorded && currentUser && feedItems) {
+                hasRenderMetricBeenRecorded = true;
```

Comment:
Some of the internals of this `if` are duplicated, and can be abstracted away to a logger/timer utility function